### ISSUE TITLE
GPUImageMovie is not available for Mac Client

### ISSUE THREAD

**Issue by osinoleg** [BradLarson/GPUImage#2035]
GPUImageMovie.h/.m don't compile (not included) for the Mac version.

**Comment by BradLarson** [comment]
Right, this class has not been ported across yet. That should be reasonably easy to do, if you want to try this yourself. Much of the AV Foundation logic will remain the same.

**Comment by tar500** [comment]
I've encountered the same issue and trying to figure out how to do it myself.

I see that CADisplayLink can be replaced with CVDisplayLinkRef, but it's hard to me to figure out how to deal with the code that is just missing from dependencies.

For example, there is no such thing as color conversion constants in GPUImageAVCamera. There is no coreVideoTextureCache in GPUImageContext on OS X. And I am even not sure yet if these things are need to be present there.

Could anyone please help me and the community with porting GPUImageMovie class to OS X?

I've made the first step, added some conditional compilation macros and some basic code, here is my changes https://github.com/tar500/GPUImage/commit/fad4abd129f2b99ba2b1bd45a14df122ecd90235

Any help would be greatly appreciated!

**Comment by tar500** [comment]
I've ported it to OS X and submitted a pull request https://github.com/BradLarson/GPUImage/pull/2122
